test/functional/android/applications_tests.py

```diff
@@ -55,6 +55,7 @@ def test_app_management(self) -> None:
         app_id = self.driver.current_package
         assert self.driver.query_app_state(app_id) == ApplicationState.RUNNING_IN_FOREGROUND
         self.driver.background_app(-1)
+        sleep(1)  # to wait the app has gone
```

Comment:
Here comes from https://github.com/appium/ruby_lib_core/blame/e13c58fe4355dcb2631f0a42248d991165f428f7/test/functional/android/android/device_test.rb#L87.

(but I'll try to avoid `sleep`.)

Reply:
Here need to wait 1 sec even after the element is invisible.